### PR TITLE
Updates the caption of gusshawstewart.md

### PR DESCRIPTION
**Pull request by gusshawstewart**
TL;DR This PR updates the copy of gusshawstewart.md because it was out of date

This PR closes / fixes / resolves #1 

## Done
- made a commit
- created a branch

## To do
- [ ] merge the branch
- [ ] delete the branch afterwards

cc. @ppremk

### CODE REVIEW

**Comment by ppremk** [comment]
@gusshawstewart is outofdate 😝

File: _slides/02-gusshawstewart.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Scotland FC are the best team in the world!!!!! not really...
```
Comment (Lowell7im) [inline _slides/02-gusshawstewart.md:6]:
```suggestion
Scotland FC are the best team in the world!!!!! not really... I agree
```

File: _slides/02-gusshawstewart.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (Lowell7im) [inline _slides/02-gusshawstewart.md:3]: This looks good

**Review by Lowell7im — APPROVED** [review]
All good. Go ahead with your changes

**Pull request closed without merging**